openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -71,6 +79,7 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
         use_asymmetric_key=use_asymmetric_key,
         is_restricted=oauth_adapter.is_client_restricted(client),
         filters=oauth_adapter.get_authorization_filters(client),
+        grant_type=grant_type,
```

Comment:
What is this change for?

Reply:
As you said I moved this `grant_type = access_token.application.authorization_grant_type` in try catch above and added grant type here.